README.md

```diff
@@ -1,2 +1,243 @@
 # QuietSpace
-A co-working space located at 16 Palmers Rd, London, E2 0SY, United Kingdom
+
+> _Where London's most creative people go to get (["deep"](https://www.youtube.com/watch?v=KX_2a_jsGYw)) work **done**_.
+
+![quiet-please-wall-art](https://cloud.githubusercontent.com/assets/194400/23511482/1cbe7720-ff55-11e6-8f21-82753e5dab6a.jpg)
+
+## _Why_?
+
+There are _hundreds_ of co-working spaces/offices in London,
+so why do we **_need_ another one**?
+
+> `As a` ***creative person*** (_writer/designer/developer/videographer_)<br />
+> `I need` a **_quite_ place** to get my **work _done_** (_without interruptions_) <br />
+> `So that` I can ***be/feel productive***. <br />
+
+> As the members of QuietSpace want to _share_ that environment with other like-minded creative people.
+
+### `QuietSpace`'s _Mission_ is to curate the most <br />_creatively productive_ working environment in London.
+
+## What?
+
+### A Co-Working Space Located at 16 Palmers Rd, London, E2 0SY
+
+Map: https://www.google.co.uk/maps/place/16+Palmers+Rd,+London+E2+0SY/ <br />
+Contact: <a href="tel:02036090609">0203 609 0609</a>
+
+#### Features
+
++ A well-designed work space with all the amenities you expect (_good internet and cell phone signal_)
```

Comment:
Remove it.
Please think of a "_hygiene factor_" https://en.wikipedia.org/wiki/Two-factor_theory
or... we can just _imply_ that _everything is awesome_.